### PR TITLE
Have log use debug instead of console

### PR DESCRIPTION
**Pull request by chadhietala**

### CODE REVIEW

File: lib/log.js [hunk 1]
```diff
@@ -32,25 +32,30 @@ module.exports = function log(tree, _options) {
       return readTree(tree).then(function(dir) {
         var label = chalk.bold.cyan;
 
-        if (options.output === 'tree') {
-          var printTree = treeOutput(dir);
+        if (process.env.DEBUG === 'true') {
```
Comment (stefanpenner) [inline lib/log.js]: this should actually just be `if (process.env.DEBUG && new RegExp(process.env.DEBUG).test(label))` as DEBUG can be a matching string

see: https://github.com/visionmedia/debug#wildcards

**Comment by stefanpenner** [comment]
thanks :)

**Comment by timmfin** [comment]
Took me a little while to figure out that I had to set `DEBUG=.*` for `stew.log` to work (after this change).

Maybe the condition should look like this instead?

```js
if (!process.env.DEBUG || new RegExp(process.env.DEBUG).test(options.label)) { ... }
```

**Comment by stefanpenner** [comment]
I think label should exist independent of this.

if `options.debugOnly` is true, that logger will only be enabled if `env.DEBUG` matches `options.label`

**Pull request merged**